Applications/TextEditor/TextEditorWidget.cpp

```diff
@@ -300,7 +300,7 @@ TextEditorWidget::TextEditorWidget()
 
     m_statusbar = add<GUI::StatusBar>();
 
-    m_editor->on_cursor_change = [update_statusbar_cursor_position = move(update_statusbar_cursor_position)] {
```

Comment:
Well, a capture-move actually does move and not copy it. So the change for `on_link_hover` is correct, but I would keep the `on_cursor_change`. Or better yet, make `update_statusbar_cursor_position` a member function.

Reply:
> make update_statusbar_cursor_position a member function.

Will do!

> Well, a capture-move actually does move and not copy it.

Can you elaborate this a bit more? I feel like I don't have a good enough grasp on the move-semantics of a closure. (Which, here, happens to be in the capture of another closure.)

Reply:
Well, that's just moving a capture into the closure rather than copying it. That was a big problem in C++11 where you couldn't move objects that were movable but not copyable into closure captures. Think of types that e.g. represent file descriptors, where such classes may be movable, but probably not copyable. This move assignment for captures allows you to take such objects and move them into a lambda.

Reply:
And such lambdas then obviously would only be movable, not copyable.

Reply:
Done ^^